components/crypto_exchange/browser/crypto_exchange_region_util.cc

```diff
@@ -0,0 +1,34 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <string>
+#include <vector>
+
+#include "brave/components/crypto_exchange/browser/crypto_exchange_region_util.h"
+
+#include "chrome/browser/profiles/profile.h"
+#include "components/country_codes/country_codes.h"
+
+namespace crypto_exchange {
+
+bool IsRegionSupported(Profile* profile, std::vector<std::string> regions,
```

Comment:
This should be updated to take in `PrefService*` as opposed to the whole profile

Reply:
This now accepts a `PrefService` item